.env.example

```diff
@@ -1,19 +1,20 @@
 # DB
 POSTGRES_USER=navigator
 POSTGRES_PASSWORD=password
-ADMIN_POSTGRES_HOST=admin_backend_db
+ADMIN_POSTGRES_HOST=admin_backend_db # for running tests outside docker this needs to be localhost
 PGUSER=navigator
 PGPASSWORD=password
-PGHOST=admin_backend_db
+PGHOST=admin_backend_db # for running tests outside docker this needs to be localhost
 
 # API
 SECRET_KEY=secret_test_key
 API_HOST=http://navigator-admin-backend:8888
 
 # AWS
```

Comment:
We could update this comment to make it more explicit/descriptive about what these are used for? Are all of these only used for local development/testing? Or are some used by the actual application?

Reply:
As someone who has not worked on bulk import, it is unclear what these are for and where/when they are used.